libraries/holochain-service-client/src/main/java/org/holochain/androidserviceruntime/holochain_service_client/IHolochainServiceCallbackStub.kt

```diff
@@ -0,0 +1,12 @@
+package org.holochain.androidserviceruntime.holochain_service_client
+
+open class IHolochainServiceCallbackStub : IHolochainServiceCallback.Stub() {
```

Comment:
Stub that provides a default implementation of the IHolochainServiceCallback interface, with no-ops for all functions.

This lets us only define a single function in each instance.